### ISSUE TITLE
is balloon dataset yours?

### ISSUE THREAD

**Issue by epsilon-deltta** [matterport/Mask_RCNN#2621]
i want to use balloon dataset in my paper. 
if this came from this repo, i have to cite this paper[1], right?

[1] Abdulla, W. (2017). Mask r-cnn for object detection and instance segmentation on keras and tensorflow.
